homeworks/sargis_vardanyan/homework1/bitwiseop/BitwiseOperators.java

```diff
@@ -0,0 +1,62 @@
+package bitwiseop;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class BitwiseOperators {
+    //Task https://www.hackerrank.com/challenges/bitwise-operators-in-c/problem?h_r=internal-search&isFullScreen=true
```

Comment:
it's great that you have link here, but you can already get yourself familiar with javadoc comments and you javadoc format to add links